internal/pkg/agent/cmd/enroll.go

```diff
@@ -351,6 +353,20 @@ func enroll(streams *cli.IOStreams, cmd *cobra.Command) error {
 
 	fromInstall, _ := cmd.Flags().GetBool(fromInstallArg)
 
+	hasRoot, err := utils.HasRoot()
+	if err != nil {
+		return fmt.Errorf("checking if running with root/Administrator privileges: %w", err)
+	}
+	if hasRoot && !fromInstall {
+		isEnroll, err := isEnroll()
+		if err != nil {
+			return fmt.Errorf("ran into an error while figuring out if user is allowed to execute the enroll command")
+		}
+		if !isEnroll {
+			return UserOwnerMismatchError
```

Comment:
On Windows you cannot actually open a shell as the `elastic-agent-user` so how would they be able to run enroll command on Windows as the `elastic-agent-user`.

Reply:
I can return an error message stating the fact that user cannot execute the command as admin for windows. Linux/mac error message will can still mention running the command as the elastic-agent-user